dhis-2/dhis-api/src/main/java/org/hisp/dhis/programrule/ProgramRuleActionType.java

```diff
@@ -70,11 +70,16 @@
         .add( HIDEFIELD, SETMANDATORYFIELD, HIDEOPTION,
             HIDEOPTIONGROUP, SHOWOPTIONGROUP )
         .build(); // Actions associated with DataElement Or
-                  // TrackedEntityAttribute
+    // TrackedEntityAttribute
 
```

Comment:
Use `ImmutableSet` as interface type, prevents caller to use `add` and get a runtime error.